subprojects/kotlin-dsl-plugins/src/integTest/kotlin/org/gradle/kotlin/dsl/plugins/dsl/KotlinCompilerWarningsTest.kt

```diff
@@ -99,7 +99,7 @@ class KotlinCompilerWarningsTest : AbstractPluginTest() {
         )
         withKotlinSourceFile()
 
-        val result = build("compileKotlin")
```

Comment:
What is this change?

Reply:
The test was actually wrong. The compilation fails because `allWarningsAsErrors = true`